src/devices/cpu/sh/sh7604.cpp

```diff
@@ -1,25 +1,310 @@
 // license:BSD-3-Clause
-// copyright-holders:R. Belmont
+// copyright-holders:Juergen Buchmueller, R. Belmont, Angelo Salese, ...
```

Comment:
The ellipsis here doesn’t look good…

Reply:
The ones with ellipsis are the credits I want to go back to double check if I missed any obvious (major) contributors. I'm not sure how strict I should be on the copyright holders list. Would it be better to just let anyone who contributed add themselves if they wanted at this point?

Reply:
Well, you know copyright rules – it has to be a substantial, creative contribution for copyright claims.  I wouldn’t worry too much about finding everyone who’s made any contribution at all.

Reply:
Ok. MAME's been using git for a while now so if someone within the past decade or 2 has made any substantial contributions and wants their name added then it should be easily verifiable at least.

Reply:
Ok, changes made to the credits to remove you from sh7604 and made you the copyright holder for sh7021 and sh7032.